llvm/lib/LTO/LTOCodeGenerator.cpp

```diff
@@ -419,7 +419,7 @@ bool LTOCodeGenerator::determineTarget() {
       Config.CPU = "apple-a12";
     else if (Triple.getArch() == llvm::Triple::aarch64 ||
              Triple.getArch() == llvm::Triple::aarch64_32)
-      Config.CPU = "cyclone";
+      Config.CPU = "apple-a7";
```

Comment:
Note that for ARM64 macOS the default CPU is `apple-m1` instead. While you are here, it would probably be worth adjusting that also (see https://github.com/llvm/llvm-project/blob/main/clang/lib/Driver/ToolChains/Arch/AArch64.cpp#L50)

There probably should be a helper function to return the default CPU string and have it used at least for all `llvm/lib/LTO` changes, possibly also in Clang, to prevent those things from drifting apart in the future again.